weed/storage/store.go

```diff
@@ -40,6 +41,7 @@ type Store struct {
 	GrpcPort            int
 	PublicUrl           string
 	Locations           []*DiskLocation
+	LocationUUIDs       []string
```

Comment:
can the locationUUID be part of the DiskLocation?

Reply:
done